packages/core/admin/admin/src/content-manager/pages/App/index.js

```diff
@@ -28,6 +33,11 @@ const App = () => {
   );
   const { pathname } = useLocation();
   const { formatMessage } = useIntl();
+  const { startSection } = useGuidedTour();
+
+  useEffect(() => {
+    startSection('contentManager');
```

Comment:
Are we sure it's safe to add startSection as a dependency?

Reply:
Wrapped it with useCallback to prevent rerenders